docs/modules/ROOT/pages/whats-new.adoc

```diff
@@ -0,0 +1,37 @@
+= What's New in Hazelcast Platform Operator
+:description: Here are the highlights of what's new and improved in Hazelcast Platform Operator 5.13.0.
+
+{description}
+
+NOTE: For new and improved features in Hazelcast Platform 5.5, see xref:{page-latest-supported-hazelcast}@hazelcast:ROOT:whats-new.adoc[What's new in Hazelcast Platform 5.5].
+
+== Community Edition and Repository Changes
+
+In March 2024, we made some updates to the Community Edition of the Hazelcast Platform, including making the Platform Operator 5.13 closed-source and its cluster creation capabilities will no longer be available in the Community Edition. 
+
+You can find more information here: https://hazelcast.com/blog/changes-to-community-edition/
```

Comment:
```suggestion
In March 2024 we made some updates to the Community Edition of Hazelcast Platform, including making Hazelcast Platform Operator 5.13 closed source, which means its cluster creation capabilities will no longer be available in the Community Edition. 

You can find more information here: https://hazelcast.com/blog/changes-to-community-edition/.
```

Reply:
Makin repo closed-source and removing open source cluster creation capabilities are two seperate things so I think we should not use `which means` in here

Reply:
I have addressed other parts with https://github.com/hazelcast/hazelcast-platform-operator-docs/pull/228/commits/39051d12189c8536137a6a807913ea8e399f2dc2

Reply:
Ok, suggested another version